### ISSUE TITLE
[BUG] Error when a player connect first on the world

### ISSUE THREAD

**Issue by uniwars** [Larkinabout/fvtt-token-action-hud-core#185]
**Describe the bug**
If a players connect first (I think, I tested on multiple worlds and it seems to be that), he will get an error of lacking permission.
![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/65986545/ba03aff5-ee69-41e0-88e5-9b30d8f9d6fc)
Players has permission to upload files

**Versions:**
- Token Action HUD Core: 1.4.17
- Token Action HUD System: Pf2e 1.4.13
- System: Pathfinder second edition  5.2.3
- Foundry VTT: v11 306
- Browser: [e.g. Google Chrome last version

**Comment by Larkinabout** [comment]
I'm not able to reproduce this issue. Can you open up the browser console and what the source of that error is?

**Comment by uniwars** [comment]
![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/65986545/935bbab9-d2b4-47ee-908a-b267c4762afe)
![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/65986545/20c4c3a9-1503-412f-8a0f-a2c44dea9b58)
![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/65986545/f50f3746-8e3b-4a5d-846f-be2a2931c63e)

**Comment by Larkinabout** [comment]
Do the errors go any further back as I don't see where Token Action HUD is involved with this error?

**Comment by uniwars** [comment]
No, and returning to setup to relaunch world doesn't make the error come back, it seems only a server reboot does, the error appeared just after updating token action hud

it doesn't seems to break anything, so it's just an error at launch, not a big deal
